presto-hive/src/test/java/com/facebook/presto/hive/util/TestStatistics.java

```diff
@@ -43,4 +51,104 @@ public void testReduce()
                 new HiveBasicStatistics(1, 2, 3, 4), SUBTRACT))
                 .isEqualTo(new HiveBasicStatistics(10, 7, 4, 1));
     }
+
+    @Test
+    public void testMergeHiveColumnStatistics()
+    {
+        assertMergeHiveColumnStatistics(
```

Comment:
I think this method actually hurts readability, since it's not clear what is happening or which is the expected output. Try inlining it.

Reply:
The idea of this method is to check that the merge is commutative. No idea how did i end up with a single assert within.